sql/v2/test/tck/binary_comparison_operators.yaml

```diff
@@ -56,6 +64,10 @@ tests:
   - name: abc is not equal to abc (diamond operator)
     expression: "'abc' <> 'abc'"
     result: false
+  - name: Diamond operator returns false when encountering a missing attribute
```

Comment:
Funny, never heard it called "Diamond operator" before. I like it! Now I feel like I've been missing out all of these years.